sdk/identity/tests.yml

```diff
@@ -10,26 +10,24 @@ resources:
 extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
-    AdditionalPlatforms:
-      Linux_NetCore_Keyring:
-        Pool: azsdk-pool-mms-ubuntu-1804-general
-        OSVmImage: MMSUbuntu18.04
-        TestTargetFramework: 'netcoreapp2.1'
-        Container: 'ubuntu_netcore_keyring'
-        SupportedClouds: 'Public,Canary,Preview'
+    AdditionalMatrixConfigs:
+      - Name: identity_container
+        Path: sdk/identity/platform-matrix.json
+        Selection: all
```

Comment:
I'd suggest sparse here as well. While we only have one right now it might be more later and we don't want to always run all of them.

Reply:
Good point, especially if people copy/paste from this one.